renovate.json

```diff
@@ -9,6 +9,9 @@
       "enabled": true
     }
   },
+  "constraints": {
+    "python": "3.7"
+  },
```

Comment:
Not sure why we're adding a constraint here -- if anything we should aim for Python 3.10 -- that said default Renovate config should be fine.

If this is about https://github.com/GoogleCloudPlatform/microservices-demo/pull/982#issuecomment-1265442132, the plan is to deprecate Cloud Debugger on our side, thus allowing us to upgrade to Python 3.10.

I suggest to close this PR. @NimJay ?